sopel/bot.py

```
@@ -777,3 +788,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
         if reply_to is None:
             reply_to = self._trigger.nick
         self._bot.reply(message, destination, reply_to, notice)
+
+    def kick(self, nick, channel=None, message=None):
+        if channel is None:
+            if self._trigger.is_privmsg:
+                return
+            else:
+                channel = self._trigger.sender
+        if nick is None:
+            return
```

This should raise an error.

---

What error would you like it to raise?

---

Same as above, `RuntimeError`.